README.md

```diff
@@ -21,51 +21,53 @@ Here are some of the benefits of using this library:
+Dynamic features can have values who's type range from object, to string, to integer and so on. Additionally, dynamic features can have an unlimited amount of values. A developer is free to choose what type should be returned when the value of a dynamic feature is requested. They are also free to choose how many options, known as variants, are available to select from.
+
+### Feature Variants
+Feature variants are the different versions of a feature that could be returned when the value of a dynamic feature is requested. Beyond the value of the feature, a feature variant contains information describing under what circumstances it should be returned over other available variants.
+
+### Feature Variant Assigners
+A feature variant assigner is a component that uses contextual information within an application to decide which feature variant should be chosen when a variant of a dynamic feature is requested.
+
 ## Registration
 
 The .NET Core configuration system is used to determine the state of feature flags. The foundation of this system is `IConfiguration`. Any provider for IConfiguration can be used as the feature state provider for the feature flag library. This enables scenarios ranging from appsettings.json to Azure App Configuration and more.
 
 ### Feature Flag Declaration
 
-The feature management library supports appsettings.json as a feature flag source since it is a provider for .NET Core's IConfiguration system. Below we have an example of the format used to set up feature flags in a json file.
+The feature management library supports appsettings.json as a feature flag source since it is a provider for .NET Core's IConfiguration system. Below we have an example of the format used to set up feature flags in a json file. The example below uses the v3 configuration schema which is supported in Microsoft.FeatureManagement version 3. For previous schemas see the configuration [schema details](./docs/schemas/README.md).
 
 ``` JavaScript
 {
-    "Logging": {
-        "LogLevel": {
-            "Default": "Warning"
-        }
-    },
-
-    // Define feature flags in a json file
+    // Define feature flags in a json configuration file
     "FeatureManagement": {
-        "FeatureT": {
-            "EnabledFor": [
-                {
-                    "Name": "AlwaysOn"
-                }
-            ]
-        },
-        "FeatureU": {
-            "EnabledFor": []
-        },
-        "FeatureV": {
-            "EnabledFor": [
-                {
-                    "Name": "TimeWindow",
-                    "Parameters": {
-                        "Start": "Wed, 01 May 2019 13:59:59 GMT",
-                        "End": "Mon, 01 July 2019 00:00:00 GMT"
+        "FeatureFlags": {
+            "FeatureT": {
+                "EnabledFor": [
+                    {
+                        "Name": "AlwaysOn"
                     }
-                }
-            ]
+                ]
+            },
+            "FeatureU": {

```

Comment:
The description under this section still talks about three features (T,U,V), but the new example only has two.